### DISCUSSION TITLE
Please make sure  php-psr extension is installed.

### DISCUSSION THREAD

**Discussion by lieshanhe** [kreait/firebase-php#602] [General]
Hello there, 

I have implemented firebase for my site recently, but I got error: PHP Fatal error: Uncaught Error: Class 'Kreait\\Firebase\\Factory' not found"

and I address it by install php-psr exteion

I posted answer here:

https://stackoverflow.com/questions/48023499/fatal-error-uncaught-error-class-kreait-firebase-serviceaccount-not-found-in/67393766#67393766

**Comment by jeromegamez** [comment]
For everyone reading this, please note that this is recommendation should not be followed without further information on the environment this applies to (system- and project-wise). A class not being found can have multiple reasons, none of which *I* know warrant the installation of a PECL extension that I never heard of until today 😅 

Also, as of today, the StackOverflow question you've linked and answered to is more than three years old.
